pkg/api/v1/schema.go

```diff
@@ -32,6 +32,22 @@ const (
 	IOThreadsPolicyAuto   IOThreadsPolicy = "auto"
 )
 
+// Represents the firmware blob used to assist in the domain creation process.
+// Used by Xen fully virtualized domains as well as setting the QEMU BIOS file path for QEMU/KVM domains
+// ---
+// +k8s:openapi-gen=true
+type Bootloader struct {
```

Comment:
once we decide on exactly how the public api should look for bootloader, we'll need to add logic to the validation webhook that ensures the users can only introduce a VMI spec with supported bootloader values.

Reply:
Is that in a different file? I assume so?

Reply:
yep, it's pretty straight forward actually. Just involves modifying these two files. 

pkg/virt-api/webhooks/validating-webhook/validating-webhook.go
pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go